enclave/src/integration-test/java/org/hyperledger/besu/enclave/EnclaveTest.java

```diff
@@ -182,8 +182,6 @@ public void testCreateFindDeleteFindPrivacyGroup() {
 
   @Test
   public void whenUpCheckFailsThrows() {
-    final Throwable thrown = catchThrowable(() -> new Enclave(URI.create("http://null")).upCheck());
-    assertThat(thrown).isInstanceOf(IOException.class);
-    assertThat(thrown).hasMessageContaining("Failed to perform upcheck");
+    assertThat(new Enclave(URI.create("http://127.0.0.1:65535")).upCheck()).isFalse();
```

Comment:
The name of this test no longer matches what it tests (i.e. no longer asserting an exception is thrown when the up check fails).

Reply:
done